include/9on12draw.inl

```diff
@@ -117,6 +118,7 @@ namespace D3D9on12
         UINT const stream0Stride = pDevice->GetPipelineState().GetInputAssembly().GetStream0Stride();
         Check9on12(stream0Stride > 0);
 
+        pDevice->SetDrawingPreTransformedVerts(true);

```

Comment:
I have a very slight preference for embedding this in the function signature for `DrawProlog`, because it makes it harder to miss other potential call sites where a draw is happening that needs to update this state. I.e. are these 4 the only types of draws?

Reply:
I think it makes sense to have it called where it is since those are the APIENTRY points for drawing. The other functions that call DrawProlog are the DrawTriangleFan functions, but those aren't actually APIENTRY points. In order for those functions to know whether they are drawing preTransformed verts we would have to pass along an extra boolean to each function anyway.

Reply:
Yeah, that's fair.